xml/ssh_banners.xml

```diff
@@ -2110,7 +2110,6 @@
     <param pos="0" name="os.product" value="Data ONTAP"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:netapp:data_ontap:-"/>
     <param pos="0" name="hw.vendor" value="NetApp"/>
-    <param pos="0" name="hw.device" value="File server"/>
```

Comment:
Also from the PR #334 - Here and elsewhere I have removed `File Server`.  This would be a new value for `hw.device` and, with these fingerprints, you can't tell if the device is a SAN, NAS, FCoIP, etc.

Also CC @hdm

Reply:
NAS is certainly a better default these days for ONTAP

Reply:
Groovy, I'm comfortable with that label as well.